CHANGELOG.md

```diff
@@ -1,3 +1,67 @@
+* Support multiple image matches for [Finding and Interacting with Image Elements](http://appium.io/docs/en/advanced-concepts/image-elements/) [appium-base-driver#449](https://github.com/appium/appium-base-driver/pull/449)
+
+### Android General
+
+* Disable to sending of Google analytics data by instruments [https://github.com/appium/appium-uiautomator2-driver/pull/452](appium-uiautomator2-driver#452), [appium-espresso-driver#666](https://github.com/appium/appium-espresso-driver/pull/666)
+* Improve to detect screen lock status [appium/appium-adb#564](https://github.com/appium/appium-adb/pull/564)
+* Add `speed` poptional arameter to mock location [appium-android-driver#681](https://github.com/appium/appium-android-driver/pull/681)
+* Fix `autoLaunch` to install applications by `app` and `otherApps` capabilities, but does not launch them as same as iOS [appium-uiautomator2-driver#440](https://github.com/appium/appium-uiautomator2-driver/pull/440), [appium-espresso-driver#654](https://github.com/appium/appium-espresso-driver/pull/654), [appium-android-driver#676](https://github.com/appium/appium-android-driver/pull/676)
+* Fix to handle special characters by `mobile:type` [appium-adb#567](https://github.com/appium/appium-adb/pull/567)
+* Fix to switch to Web Views with same package name, but different process names [appium-android-driver#687](https://github.com/appium/appium-android-driver/pull/687)
+
+### Android(UIAutomator2)
+
+* Add `mobile:` functions:
+    * `mobile:doubleClickGesture` for double click guesture [appium-uiautomator2-driver#441](https://github.com/appium/appium-uiautomator2-driver/pull/441), [appium-uiautomator2-server#415](https://github.com/appium/appium-uiautomator2-server/pull/415)
+    * `mobile:installMultipleApks` to install multiple apks via `adb install-multiple` command. Please read [mobile:installMultipleApks](https://github.com/appium/appium-uiautomator2-driver/blob/master/README.md#mobile-installmultipleapks) for more details
+* Fix XPath lookup for queries which have `@*` attribute pattern [appium-uiautomator2-server#417](https://github.com/appium/appium-uiautomator2-server/pull/417)
+* Fix to stop screen recording after a session is closed [appium-uiautomator2-driver#439](https://github.com/appium/appium-uiautomator2-driver/pull/439)
+
+### Android(Espresso)
+
+* Fix to raise an error in [Reset App](http://appium.io/docs/en/commands/device/app/reset-app/). It must quit the session and create a new one in Espresso driver. [appium-espresso-driver#649](https://github.com/appium/appium-espresso-driver/pull/649)
+* Fix to reset instrumentation state before starting a new instance [appium-espresso-driver#662](https://github.com/appium/appium-espresso-driver/pull/662)
+* Gradle wrapper has been rewritten using Kotlin DSL [appium-espresso-driver#658](https://github.com/appium/appium-espresso-driver/pull/658), [appium-espresso-driver#659](https://github.com/appium/appium-espresso-driver/pull/659)
```

Comment:
I would say it makes sense to explicitly mention people who don't belong to the team and who dedicated their time to improve Appium here